xml/tuning_storagescheduler.xml

```diff
@@ -83,9 +83,19 @@
   <note os="sles" arch="x86_64">
    <title>Scheduler in case of block multi-queue (blk-mq) I/O path</title>
    <para>
-     If the device is using blk-mq I/O path (refer to <xref
-     linkend="cha-tuning-io-scsimq"/>)
-     <replaceable>SCHEDULER</replaceable> should be set to either
+     The <literal>elevator</literal> boot parameter does not apply to
+     devices using blk-mq I/O path (refer to <xref
+     linkend="cha-tuning-io-scsimq"/>).
+   </para>
+   <para>
+     Default elevator is <option>mq-deadline</option> for conventional
+     devices (e.g. rotational hard disks, SSDs) and
```

Comment:
```suggestion
     devices (for example, rotational hard disks, SSDs) and
```

Usually we use the term _spinning hard disks_. The term _rotational_ is new to me. Feel free to keep it if it's an official term. :-)